docs/guides/index.mdx

```diff
@@ -381,7 +381,7 @@ This returns the `playbackURL` at the bottom of the response which we will use t
 ##  Playback a Livestream
 
 The response from requesting an asset in the API will return the `playbackId` which is used for the `playbackUrl`. https://livepeercdn.com/hls/{playbackId}/index.m3u8
-> **Note:** You can also get the `playbackUrl` by from the dashboard of the stream.
+> **Note:** You can also get the `playbackUrl` from the dashboard of the stream.
 
 
 There are different options to view a livestream.
```

Comment:
`Click the  `Load Settings` button when you press `play`.` doesn't make sense. Can we re-word?